src/App.js

```diff
@@ -1,84 +1,99 @@
 import React, { Component } from 'react';
 import './App.css';
-
-import Home from './component/Home';
-import Completed from './component/Completed';
-import Remaining from './component/Remaining';
+import Container from './component/Container';
+import AddTodoInput from './component/AddTodoInput';
+import Header from './component/Header';
+import Tabs from './component/Tabs';
 
 class App extends Component {
+
   constructor() {
     super();
     this.state = {
-      todoList: [
-        {
-          id: 123,
-          content: 'create react app',
-          completed: false
-        },
-        {
-          id: 124,
-          content: 'create java app',
-          completed: true
-        }
-      ],
+      todoList: [],
       currentView: 'home'
     };
   }
 
   render() {
-    let currentView = this.state.currentView;
-
     return (
-      <div>
-        {/* Header */}
-        <div>
-          {/* tabs */}
-          <div>
-            <ul>
-              <li onClick={() => this.setState({ currentView: 'home' })}>
-                Home
-              </li>
-              <li onClick={() => this.setState({ currentView: 'completed' })}>
-                Completed
-              </li>
-              <li onClick={() => this.setState({ currentView: 'remaining' })}>
-                Remaining
-              </li>
-            </ul>
-          </div>
+      <div className='main-container__div'>
+        <div className='header_container__div'>
+          {/* Header */}
+          <Header />
```

Comment:
Since the component name is very explicit we can get rid of the `{/* Header */}` comment. Remove the comments below as well.